packages/web/utils/formatter.ts

```diff
@@ -5,21 +5,37 @@ import {
   PricePretty,
   RatePretty,
 } from "@keplr-wallet/unit";
+import { trimZerosFromEnd } from "@osmosis-labs/stores";
+
+type FormatOptions = Partial<
+  Intl.NumberFormatOptions & { maxDecimals: number }
+>;
+
+type FormatOptionsWithDefaults = Partial<Intl.NumberFormatOptions> & {
+  maxDecimals: number;
+};
+
+const DEFAULT = {
+  maxDecimals: 2,
+};
 
 /** Formats a pretty object as compact by default. i.e. $7.53M or $265K, or 2K%. Validate handled by pretty object. */
 export function formatPretty(
   prettyValue: PricePretty | CoinPretty | RatePretty | Dec,
-  opts?: Partial<Intl.NumberFormatOptions>
+  opts: FormatOptions = {}
 ) {
-  const { ...formatOpts } = opts || {};
+  const optsWithDefaults: FormatOptionsWithDefaults = {
+    ...{ ...DEFAULT },
```

Comment:
```suggestion
    ...DEFAULT,
```

Reply:
We should probably add a comment to prevent future removal of this section. At a glance, it appears to be an accidental duplicate clone.